src/main/java/org/broadinstitute/hellbender/utils/IntegrationUtils.java

```diff
@@ -0,0 +1,55 @@
+package org.broadinstitute.hellbender.utils;
+
+import org.apache.commons.math3.analysis.integration.gauss.GaussIntegrator;
+import org.apache.commons.math3.analysis.integration.gauss.GaussIntegratorFactory;
+
+import java.util.Arrays;
+import java.util.function.DoubleFunction;
+import java.util.function.ToDoubleBiFunction;
+import java.util.stream.IntStream;
+
+/**
+ * Created by tsato on 5/1/17.
+ */
+public class IntegrationUtils {
+    private static final GaussIntegratorFactory integratorFactory = new GaussIntegratorFactory();
+
+    public static double integrate(final DoubleFunction<Double> getIntegrand, final double lowerBound, final double upperBound, final int numPoints) {
+        final GaussIntegrator integrator = integratorFactory.legendre(numPoints, lowerBound, upperBound);
+
+        final double[] gaussIntegrationWeights = IntStream.range(0, numPoints).mapToDouble(integrator::getWeight).toArray();
```

Comment:
A much faster way to do this (because Java streams are expensive to construct and the JVM doesn't yet recognize a simple map operation) is `final double[] gaussIntegrationWeights = new IndexRange(0, numPoints).mapToDouble(integrator::getWeight)`.